### ISSUE TITLE
Timeout exceed using odr: 10 and output_mode: 3

### ISSUE THREAD

**Issue by rebecatourinho** [xsens/xsens_mti_ros_node#37]
I'm using v4.0.0 and have noticed that timeout is not enough for odr: 10 and output_mode: 3 configuration entries on xsens.yaml. The other output modes are working fine at the same odr: 10. I'm testing a MTi-30 under xkf_scenario: 39 and baudrate: 115200 and even tried this in other computers.

Finally, I've changed additionalTimeOutOffset from "0.010" to "0.020" and it worked. Could anyone explain what is happening once output_mode 3 has fewer data than mode 1 or 2?

**Comment by arunvydhya** [comment]
@rebecatourinho  yes, mode 3 has fewer messages than the ones for 1 and 2. You can modify mtdevice.py to zero in on the payload you are interested in.
